llvm/test/CodeGen/X86/vselect.ll

```diff
@@ -7,6 +7,38 @@
 ; Verify that we don't emit packed vector shifts instructions if the
 ; condition used by the vector select is a vector of constants.
 
+define <2 x i64> @masked_select_const(<2 x i64> %a, <2 x i64> %x, <2 x i64> %y) {
```

Comment:
Use `<4 x i32>` to avoid unnecessary `bitcast`. And move the test to the end of the file since it's not directly related to above comments.